lunavl/sdk/estimators/face_estimators/mask.py

```diff
@@ -0,0 +1,166 @@
+"""Module for estimate a warped image quality.
+
+See `warp quality`_.
+"""
+from typing import Union, Dict
+
+from FaceEngine import MedicalMaskEstimation, IMedicalMaskEstimatorPtr  # pylint: disable=E0611,E0401
+
+from lunavl.sdk.errors.errors import LunaVLError
+from lunavl.sdk.errors.exceptions import CoreExceptionWrap, LunaSDKException
+from ..base import BaseEstimator
+from ..face_estimators.facewarper import FaceWarp, FaceWarpedImage
+from ...base import BaseEstimation
+
+
+class Mask(BaseEstimation):
+    """
+    Structure mask
+
+    Estimation properties:
+
+        - mask
+    """
+
+    #  pylint: disable=W0235
+    def __init__(self, mask: MedicalMaskEstimation):
+        """
+        Init.
+
+        Args:
+            mask: estimated mask
+        """
+        super().__init__(mask)
+
+    @property
+    def maskInPlace(self) -> float:
+        """
+        Get mask in place probability.
```

Comment:
It is not clear. And other comments in the class too

Reply:
updated